### DISCUSSION TITLE
Algoritmi uporedjivanja

### DISCUSSION THREAD

**Discussion by ivan-ristovic** [matf-pp/2021_Algoritam-tester#1] [Ideas]
Mozda ne bi bila losa ideja imati vise razlicitih "modova" za rad, na primer:
- raw string matching
- regex matching
- ignore whitespace matching
- itd.

**Comment by Dzondzi** [comment]
Radicemo na haskellu ipak ako je to u redu, koncept je slican onome na webgradeu odnosno kako mi zamisljamo da webgrader radi
